tests/models/clip/test_modeling_clip.py

```diff
@@ -477,7 +477,9 @@ def prepare_config_and_inputs_for_common(self):
 @require_torch
 class CLIPModelTest(ModelTesterMixin, PipelineTesterMixin, unittest.TestCase):
     all_model_classes = (CLIPModel,) if is_torch_available() else ()
-    pipeline_model_mapping = {"feature-extraction": CLIPModel} if is_torch_available() else {}
+    pipeline_model_mapping = (
+        {"feature-extraction": CLIPModel, "image-feature-extraction": CLIPVisionModel} if is_torch_available() else {}
```

Comment:
@ydshieh Is this how the pipeline mapping should be added for composite models? Or would it be better to add under `CLIPVisionModelTest` ?

Reply:
The problem is that we don't have `hf-internal-testing/tiny-random-CLIPVisionModel`: the tiny model are only created for the top level model class not their components.

Reply:
OK - I'll just remove for all the vision component models